README.md

```diff
@@ -1888,6 +1888,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Tijani Hussein](https://github.com/Shield01)
   - [Tikeshwar Sahu](https://github.com/Tiku24)
   - [Tinashe Murambinda](https://github.com/Draxeytina)
+  - [Timothy Tyler DeHof](https://github.com/timDeHof)  
```

Comment:
I think your name must be above Tinashe Murambinda.
Let us know if you need any help with updating that. 🙂

Reply:
yes, you are correct. I had a momentary brain fart on alphabet.I will move it.